Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM elixir:1.9-alpine as builder
+
+LABEL author "Emil Bostijancic <emil.bostijancic@gmail.com>"
+
+ENV MIX_ENV prod
+#ENV REPLACE_OS_VARS true
+
+RUN apk add --no-cache build-base git bash curl ca-certificates
+RUN mix local.hex --force
+RUN mix local.rebar --force
+RUN mkdir -p /mini_repo
+
+WORKDIR /mini_repo
+
+ADD config  ./config/
+ADD lib  ./lib/
+ADD priv ./priv/
+ADD rel ./rel/
+COPY mix.exs mix.lock ./
+
+RUN mix deps.get
+RUN mix compile
+RUN mix release
+
+FROM alpine:3.9
+
+RUN apk add --no-cache ncurses-libs openssl bash curl ca-certificates
```

Comment:
we may not need ncurses-libs, curl, ca-certificates - could you double-check?

Reply:
ncurses-libs in needed, otherwise the following error is thrown:
```Error loading shared library libncursesw.so.6: No such file or directory (needed by /app/erts-10.4.4/bin/beam.smp)
Error relocating /app/erts-10.4.4/bin/beam.smp: tputs: symbol not found
Error relocating /app/erts-10.4.4/bin/beam.smp: tgetflag: symbol not found
Error relocating /app/erts-10.4.4/bin/beam.smp: tgetent: symbol not found
Error relocating /app/erts-10.4.4/bin/beam.smp: tgetnum: symbol not found
Error relocating /app/erts-10.4.4/bin/beam.smp: tgetstr: symbol not found```